xilinx/examples/ultra96/blinky.tcl

```diff
@@ -0,0 +1,3 @@
+open_checkpoint blinky.dcp
+write_verilog -force blinky_out.v
+write_bitstream -force blinky.bit
```

Comment:
Please add a newline at the end of the file

Reply:
done